lib/async-git-prompt.zsh

```diff
@@ -0,0 +1,33 @@
+ASYNC_PROC=0
+function precmd() {
```

Comment:
Instead of using plain `precmd` (of which there can only be one), should be `_omz_git_async_precmd` and then added to the hooks list with `precmd_functions+=_omz_git_async_precmd`. That way it will play nicer with other things that have precmd hooks.